fsspec/tests/abstract/copy.py

```diff
@@ -102,15 +102,9 @@ def test_copy_directory_to_existing_directory(
                 assert fs.isfile(fs_join(target, "subfile2"))
                 assert fs.isdir(fs_join(target, "nesteddir"))
                 assert fs.isfile(fs_join(target, "nesteddir", "nestedfile"))
+                assert not fs.exists(fs_join(target, "subdir"))
 
-                fs.rm(
-                    [
-                        fs_join(target, "subfile1"),
-                        fs_join(target, "subfile2"),
-                        fs_join(target, "nesteddir"),
-                    ],
-                    recursive=True,
-                )
+                fs.rm(fs.ls(target, detail=False), recursive=True)
```

Comment:
Care needed using `ls` as some implementations have default `detail` of `True` and some `False`. cc https://github.com/fsspec/filesystem_spec/issues/1205

Reply:
being explicit is always good